File: usuarios-api/app/models/usuario.py
from sqlmodel import SQLModel, Field, Relationship
from typing import Optional

class Usuario(SQLModel, table=True):
    id: Optional[int] = Field(default=None, primary_key=True)
    nome: str
    email: str
    cargo: str
    id_equipe: Optional[int] = Field(default=None, foreign_key="equipe.id")

File: andamento-api/src/models/usuario.ts
import { Entity, Column, PrimaryGeneratedColumn, ManyToOne, JoinColumn, OneToMany } from 'typeorm';
import { Equipe } from './equipe';
import { Tarefas } from './tarefas';
import { Andamento } from './andamento';

@Entity('usuarios')
export class Usuario {
    @PrimaryGeneratedColumn()
    id: number;

    @Column()
    nome: string;

    @Column()
    email: string;

    @Column()
    cargo: string;

    @ManyToOne(() => Equipe, equipe => equipe.usuarios)
    @JoinColumn({ name: 'id_equipe' })
    equipe: Equipe;

    @OneToMany(() => Tarefas, tarefa => tarefa.responsavel)
    tarefasResponsaveis: Tarefas[];

    @OneToMany(() => Andamento, andamento => andamento.usuario)
    andamentos: Andamento[];

}